src/gpuarray/buffer.h

```diff
@@ -550,13 +550,7 @@ GPUARRAY_PUBLIC gpucontext *gpukernel_context(gpukernel *k);
  */
 #define GA_CTX_PROP_DEVNAME  1
 
-/**
```

Comment:
I got confused myself! So what is the name for the maximum total local size? Is it available?